### ISSUE TITLE
List options present under 'Level' combo box are not clearly visible after resizing to 200%: A11y_.NetAspireDashboard_StructuredLog_Alldropdown_Zoom

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3257]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #Benchmark; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG1.4.4; #Zoom;

### Prerequisites:
Display resolution: 1280* 768.

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: https://localhost:15887 and login with valid credentials.
2. TAB to "Structured Log" tab item and hit ENTER key.
3. TAB to "All dropdown" and hit ENTER key.
4. Zoom the page to 200% and observe the issue.

### Actual Results:
List options present under 'Level' combo box are not clearly visible after resizing to 200%.

### Expected Results:
List options present under 'Level' combo box should be clearly visible after resizing to 200%.

### User Impact:
Low vision users will not be able to see the list options clearly and will miss the information.

### Attachment:
![3257_A11y_ NetAspireDashboard_StructuredLog_Alldropdown_BeforeResize](https://github.com/dotnet/aspire/assets/90901519/c36f1497-54bf-433f-9252-a031328c2a73)
![3257_A11y_ NetAspireDashboard_StructuredLog_Alldropdown_AfterResize](https://github.com/dotnet/aspire/assets/90901519/5470ffc0-70d8-47f6-9ad8-6c125f809b2f)

**Comment by adamint** [comment]
@PoojaNamde can you retest this/give exact dimensions? I cannot reproduce this.

**Comment by PoojaNamde** [comment]
@adamint  Please set the display resolution to 1280* 768 and zoom the browser to 200% to repro this issue.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:04-22-24;

Verified the bug in environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment
![#3257_Resize200%](https://github.com/dotnet/aspire/assets/90901519/fc58e091-8e38-4206-bb77-5671ebd604c1)
